### PR TITLE
Tabbifying script examples per OS

### PR DESCRIPTION
**Pull request by CamSoper**
Adds "Windows/Other" OS tabs for the scripts.  The only thing that changes between the two is the syntax for comments, but I want to be sure a user can copy/paste with a minimum of fuss.
[Review link](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/publish-to-azure-webapp-using-cli?branch=pr-en-us-4735&tabs=windows)

### CODE REVIEW

**Comment by CamSoper** [comment]
@Rick-Anderson Good to merge if you're happy with it.  Thanks!  :)

File: aspnetcore/toc.md [hunk 1]
```diff
@@ -196,7 +196,7 @@
 #### [Publish to a Docker Image](https://azure.microsoft.com/documentation/articles/vs-azure-tools-docker-hosting-web-apps-in-docker/)
 ## [Deployment](xref:tutorials/publish-to-azure-webapp-using-vs)
 ### [Publish to Azure with Visual Studio](xref:tutorials/publish-to-azure-webapp-using-vs)
-### [Publish to Azure with CLI tools](xref:tutorials/publish-to-azure-webapp-using-cli)
+### [Publish to Azure with CLI tools](tutorials/publish-to-azure-webapp-using-cli.md)
```
Comment (Rick-Anderson) [inline aspnetcore/toc.md]: Can you use xref:
They are more robust than relative paths. You can copy/paste them.

**Comment by CamSoper** [comment]
Okay, fixed that TOC.  Good to go, @Rick-Anderson!

**Comment by Rick-Anderson** [comment]
@CamSoper  nice work, thanks!

**Pull request merged**